frontend/src/utils/queryGraph.js

```diff
@@ -57,7 +78,7 @@ const standardizeCategory = (o) => standardizeArrayProperty(o, 'category');
  * @param {object} obj object to prune
  * @param {string} property property of object to prune
 */
-function pruneEmptyArrays(obj, property) {
+function pruneEmptyArray(obj, property) {
   if (obj[property] && _.isArray(obj[property]) &&
       obj[property].length === 0) {
     delete obj[property];
```

Comment:
This syntax is crazy. `obj[property]` is essentially not an expression here, but just some familiar syntax describing what to delete. Python does the same thing.

Reply:
I'm assuming this is just a remark and not actually actionable.